sonar-plugin-api/src/main/java/org/sonar/api/batch/DecoratorBarriers.java

```diff
@@ -28,7 +28,9 @@
  * </ul>
  *
  * @since 2.3
+ * @deprecated since 5.6 no more used since Decorators are deprecated
  */
+@Deprecated
 public interface DecoratorBarriers {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)